scripts/go-tool.sh

```diff
@@ -64,12 +64,10 @@ fi
 function invoke_go() {
   tool="$1"
   shift
-  # TODO(ROX-27054): Remove the redundant strictfipsruntime option if one is found to be so
-  if [[ -z "$GOTAGS" ]]; then
-    GOTAGS="$GOTAGS,strictfipsruntime"
-  else
-    GOTAGS="strictfipsruntime"
-  fi
+
+  # ${parameter:+word}: If parameter is null or unset, nothing is substituted, otherwise the expansion of word is substituted.
+  # - https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion
+  GOTAGS="${GOTAGS:+${GOTAGS},}strictfipsruntime"
   if [[ "$RACE" == "true" ]]; then
     GOEXPERIMENT=strictfipsruntime CGO_ENABLED=1 go "$tool" -race -ldflags="${ldflags[*]}" -tags "$(tr , ' ' <<<"$GOTAGS")" "$@"
```

Comment:
I know you did not change that part but we can slightly simplify things here.
`go help build` says this:
```
        -tags tag,list
                a comma-separated list of additional build tags to consider satisfied
                during the build. For more information about build tags, see
                'go help buildconstraint'. (Earlier versions of Go used a
                space-separated list, and that form is deprecated but still recognized.)
```

Therefore, no need in `tr , ' '` any more:
```suggestion
    GOEXPERIMENT=strictfipsruntime CGO_ENABLED=1 go "$tool" -race -ldflags="${ldflags[*]}" -tags "$GOTAGS" "$@"
```